src/webapp/pages/analysis/AnalysisPage.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import i18n from "../../../utils/i18n";
+import { PageHeader } from "../../components/page-header/PageHeader";
+import { Wizard } from "@eyeseetea/d2-ui-components";
+import { steps } from "./steps";
+
+type PageProps = {
+    name: string;
+};
+
+export const AnalysisPage: React.FC<PageProps> = React.memo(props => {
+    const { name = "Analysis Page" } = props;
```

Comment:
No need to repeat the name since the value is being passed from the `Route` component.

```tsx
<Route path="/analysis" render={() => <AnalysisPage name="Analysis Page" />} />
```

```tsx
const { name } = props;
return (
        <React.Fragment>
            <PageHeader title={name} />
            <Wizard steps={steps} />
        </React.Fragment>
    );
```